packages/ui/feature-builder-right-sidebar/src/lib/edit-step-sidebar/edit-step-form-container/edit-step-form-container.component.html

```diff
@@ -16,7 +16,8 @@
               <ap-markdown [fullWidth]="true"
                 data="If you are expecting a reply from this webhook, append <b>/sync</b> to the URL.<br> <br>
                   In that case, you will also have to add an <b>HTTP step</b>  with return response to the end of your flow.<br> <br>
-                  **If the flow takes more than 30 seconds, it will give a 204 no-content response.**
+                  **If the flow takes more than 30 seconds, it will give a 204 no-content response.**<br>
+                  To change this time, set the **AP_WEBHOOK_TIMEOUT** to desired number.
```

Comment:
You can use flagService to check If it's not cloud

Reply:
You can check like that

In html:

`edition$ | async === ApEdition.CLOUD`

in TS:
`const edition$ = flagsService.getEdition()`

There might be typos, I am typing it from mobile